src/Microsoft.SqlTools.ServiceLayer/Scripting/Contracts/ScriptingErrorEvent.cs

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Scripting.Contracts
+{
+    /// <summary>
+    /// Parameters to be sent to indicate a scripting operations has encountered and error.
+    /// </summary>
+    public class ScriptingErrorParams : ScriptingEventParams
+    {
+        public string Message { get; set; }
+
+        public string DiagnosticMessage { get; set; }
+    }
+
+    /// <summary>
+    /// Event sent to indicate a scripting database operation has encountered an error.
+    /// </summary>
+    public class ScriptingErrorEvent
```

Comment:
Will do.

---
In reply to: [111651923](https://github.com/Microsoft/sqltoolsservice/pull/316#discussion_r111651923) [](ancestors = 111651923)